build.sbt

```diff
@@ -69,36 +69,36 @@ val flinkDeps =
     // sql parser
     "org.apache.flink"  % "flink-sql-parser"                     % V.flink,
     // queryable state
-    "org.apache.flink"  % "flink-queryable-state-runtime"        % V.flink % Provided,
+    "org.apache.flink"  % "flink-queryable-state-runtime"        % V.flink      % Provided,
     // complex event processing
-    "org.apache.flink"  % "flink-cep"                            % V.flink % Provided,
+    "org.apache.flink"  % "flink-cep"                            % V.flink      % Provided,
     // connectors
-    "org.apache.flink"  % "flink-connector-base"                 % V.flink % Provided, // ds hybrid source
-    "org.apache.flink"  % "flink-connector-files"                % V.flink % Provided, // ds text files
-    "org.apache.flink"  % "flink-parquet"                        % V.flink % Provided, // parquet bulk sink
-    "org.apache.flink"  % "flink-connector-kafka"                % V.flink % Provided,
-    "org.apache.flink"  % "flink-connector-kinesis"              % V.flink % Provided,
-    "org.apache.flink"  % "flink-connector-aws-kinesis-streams"  % V.flink % Provided,
-    "org.apache.flink"  % "flink-connector-aws-kinesis-firehose" % V.flink % Provided,
-    "org.apache.flink" %% "flink-connector-cassandra"            % V.flink % Provided,
-    "org.apache.flink"  % "flink-connector-elasticsearch7"       % V.flink % Provided,
-    "org.apache.flink"  % "flink-connector-jdbc"                 % V.flink % Provided,
-    "org.apache.flink"  % "flink-connector-rabbitmq"             % V.flink % Provided,
+    "org.apache.flink"  % "flink-connector-base"                 % V.flink      % Provided, // ds hybrid source
+    "org.apache.flink"  % "flink-connector-files"                % V.flink      % Provided, // ds text files
+    "org.apache.flink"  % "flink-parquet"                        % V.flink      % Provided, // parquet bulk sink
+    "org.apache.flink"  % "flink-connector-kafka"                % V.flink      % Provided,
+    "org.apache.flink"  % "flink-connector-kinesis"              % "4.1.0-1.17" % Provided,
```

Comment:
Can we replace the `1.17` here with `flinkMinor`?

Reply:
Yes

Reply:
Doing now